nautobot/core/tables.py

```diff
@@ -64,6 +64,9 @@ def __init__(
                 returns new data. Runs after all of the queryset auto-optimization performed by this class.
                 Used for example in IPAM views to inject "fake" records for "available" Prefixes, IPAddresses, or VLANs.
             **kwargs (dict, optional): Passed through to django_tables2.Table
+        .. warning::
+            Do not modify/set the `base_columns` attribute after BaseTable class is instantiated.
+            Do not modify/set the `base_columns` attribute after calling super().__init__() of BaseTable class.
```

Comment:
May need to adjust this a bit to make it render well in mkdocs (https://docs-landing-page--6670.org.readthedocs.build/projects/core/en/6670/code-reference/nautobot/apps/tables/#nautobot.apps.tables.BaseTable):

![image](https://github.com/user-attachments/assets/c183a605-278e-4df3-8f9e-6270d8408321)